ui/src/app/components/sidenav-save-session/sidenav-save-session.component.ts

```diff
@@ -38,7 +38,7 @@ export class SidenavSaveSessionComponent implements OnInit {
       SessionName: formValue.SessionName.trim(),
       EditorName: formValue.EditorName.trim(),
       DatabaseName: formValue.DatabaseName.trim(),
-      Notes: formValue.Notes.split('\n'),
```

Comment:
Not getting session exists in case of same session name. Getting the session name already exists when session name, database name is duplicate. Either we should improve the error messaging saying session x for database y exists or keep a check only on session name

Reply:
Resolved by updating error message with "session x for database y exists".